notify/impl_test.go

```diff
@@ -74,6 +74,73 @@ func TestPagerDutyRetryV2(t *testing.T) {
 	}
 }
 
+func TestPagerDutyRedactedURLV1(t *testing.T) {
+	done := make(chan struct{})
+	ctx, cancel := context.WithCancel(context.Background())
+	ctx = WithGroupKey(ctx, "1")
+
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		cancel()
+		<-done
+	}))
+	defer func() {
+		close(done)
+		srv.Close()
+	}()
+
+	key := "01234567890123456789012345678901"
+	notifier := NewPagerDuty(
+		&config.PagerdutyConfig{
+			ServiceKey: config.Secret(key),
+			HTTPConfig: &commoncfg.HTTPClientConfig{},
+		},
+		createTmpl(t),
+		log.NewNopLogger(),
+	)
+	notifier.apiV1 = srv.URL
+
+	ok, err := notifier.Notify(ctx, []*types.Alert{}...)
```

Comment:
This seems to be the same for each added test. Could this be a table-driven test with [subtests](https://blog.golang.org/subtests) instead? 

Let me know if you want to delay it, as it is not necessary for the security patch release.

Reply:
Rather than table-driven tests (to keep each test focused on one receiver), I'll create a helper function.

Reply:
Sounds good, thanks!

Reply:
Done!